container/Dockerfile.sglang

```diff
@@ -529,3 +531,70 @@ RUN python3 -m pip install --no-cache-dir \
 
 ENTRYPOINT ["/opt/nvidia/nvidia_entrypoint.sh"]
 CMD []
+
+###########################################################
+########## Runtime with AWS EFA ##########################
+###########################################################
+#
+# This stage extends the runtime stage with AWS EFA installer which includes: libfabric and aws-ofi-nccl plugin
+#
+# Use this stage when deploying on AWS infrastructure with EFA support
+
+FROM runtime AS runtime-aws
+
+ARG EFA_VERSION
+
+USER root
+
+# Install AWS EFA installer with bundled libfabric and aws-ofi-nccl
+# Flags explanation:
+#   --skip-kmod: Skip kernel module installation (handled by host)
+#   --skip-limit-conf: Skip ulimit configuration (handled by container runtime)
+#   --no-verify: Skip GPG verification (optional, can be removed if verification is needed)
+RUN mkdir -p /tmp/efa && \
+    cd /tmp/efa && \
+    curl --retry 3 --retry-delay 2 -fsSL -o aws-efa-installer-${EFA_VERSION}.tar.gz \
+        https://efa-installer.amazonaws.com/aws-efa-installer-${EFA_VERSION}.tar.gz && \
+    tar -xf aws-efa-installer-${EFA_VERSION}.tar.gz && \
+    cd aws-efa-installer && \
+    apt-get update && \
+    ./efa_installer.sh -y --skip-kmod --skip-limit-conf --no-verify && \
+    rm -rf /tmp/efa && \
+    ldconfig
+
+USER dynamo
+
+ENTRYPOINT ["/opt/nvidia/nvidia_entrypoint.sh"]
+CMD []
+
+########################################################################
```

Comment:
as a thought - is it possible to have a common aws dockerfile that would take any runtime image as base and make the changes? in order to not have to change the files directly here?